setup.py

```diff
@@ -10,15 +10,15 @@
 
 setup(
     name="mo-gym",
-    version="0.1.1",
+    version="0.2.0",
     description="Environments for Multi-Objective RL.",
     url="https://www.github.com/LucasAlegre/mo-gym",
     author="LucasAlegre",
     author_email="lnalegre@inf.ufrgs.br",
     license="MIT",
     packages=[package for package in find_packages() if package.startswith("mo_gym")],
     install_requires=[
-        "gym==0.24.1", # 0.25 has breaking changes
+        "gym>=0.26", # 0.26 has breaking changes
```

Comment:
We can remove the comment

Reply:
why?

Reply:
The comment was there to explain why we forced the version. Now I don't see any reason to keep the version forced, hence the >=0.26 and not ==0.26.